grug/app/Cargo.toml

```diff
@@ -10,7 +10,7 @@ rust-version  = { workspace = true }
 version       = { workspace = true }
 
 [features]
-abci    = ["data-encoding", "tower", "tower-abci"]
+abci    = ["data-encoding", "tendermint", "tower", "tower-abci"]
```

Comment:
technically, data-encoding should only be enabled if both `abci` and `tracing` are enabled. but i don't know to do it. in practice though, if we use `abci` we always use `tracing` so not a problem.